src/content/use-the-api/kubectl-gs/template-cluster.md

```diff
@@ -658,6 +704,112 @@ status:
   ready: false
 ```
 
+{{< /tab >}}
+{{< tab id="command-output-azure-capi" for-impl="capz_vms">}}
+
+```yaml
+---
+apiVersion: v1
+data:
+  values: |
+    connectivity:
+      bastion:
+        enabled: true
+    controlPlane:
+      replicas: 3
+    metadata:
+      description: Development Cluster
+      name: dev01
+      organization: acme
+    providerSpecific:
+      location: germanywestcentral
+      subscriptionId: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
```

Comment:
```suggestion
```

Reply:
This is what the command generates actually